www/front_src/src/components/wizardFormInstallingStatus/index.tsx

```diff
@@ -0,0 +1,96 @@
+/* eslint-disable react/jsx-filename-extension */
+/* eslint-disable react/prop-types */
+
+import React from 'react';
+
+import { useTranslation } from 'react-i18next';
+
+import { Typography, Paper, Grid, Alert } from '@mui/material';
+import Chip from '@mui/material/Chip';
+import CheckIcon from '@mui/icons-material/Check';
+import SmsFailedIcon from '@mui/icons-material/SmsFailed';
+
+import { ContentWithCircularLoading } from '@centreon/ui';
+
+import { useStylesWithProps } from '../../styles/partials/form/PollerWizardStyle';
+
+interface Props {
+  error: string | null;
+  formTitle: string;
+  statusCreating: boolean | null;
+  statusGenerating: boolean | null;
+}
+
+export default ({
+  formTitle,
+  statusCreating,
+  statusGenerating,
+  error,
+}: Props): JSX.Element => {
+  const classes = useStylesWithProps({ statusCreating, statusGenerating });
+
+  const { t } = useTranslation();
+  const loading = statusCreating === null || statusGenerating === null;
+  const hasError =
+    (statusCreating === false || statusGenerating === false) && error;
```

Comment:
```suggestion
  const loading = isNil(statusCreating) || sNil(statusGenerating);
  const hasError =
    (equals(statusCreating, false) || equals(statusGenerating, false)) && error;
```

isNil and equals are imported from ramda https://ramdajs.com/docs/